pkg/kritis/apis/kritis/v1beta1/attestationauthority.go

```diff
@@ -33,9 +39,37 @@ type AttestationAuthority struct {
 
 // AttestationAuthoritySpec is the spec for a AttestationAuthority resource
 type AttestationAuthoritySpec struct {
-	NoteReference string   `json:"noteReference"`
-	PublicKeyList []string `json:"publicKeyList"`
-	PolicyType    string   `json:"policyType"`
+	NoteReference string      `json:"noteReference"`
+	PublicKeys    []PublicKey `json:"publicKeys"`
+	PolicyType    string      `json:"policyType"`
+}
+
+// PublicKey stores key data used to verify Attestations.
+type PublicKey struct {
+	// KeyId is the ID of this public key (required for PKIX keys). For PKIX
```

Comment:
After our discussion with Nenad, we decided that the `keyId` field would be required for PKIX keys. We are unable to achieve full compatibility with Binauthz in this regard, because Kritis lacks a policy management service (PMS). 

Binauthz's PMS can create its own PKIX `keyId` that it can return to the users. Since Kritis has no such system, we must ask that our users define and refer to their keys through an ID that they provide.